requirements-all.txt

```diff
@@ -6,7 +6,7 @@
 -r requirements-apps-update-db.txt
 boto3==1.24.70
 jinja2==3.1.2
-moto==1.3.14
+moto[dynamodb]==4.0.2
```

Comment:
Just out of curiosity what does the `[dynamodb]` do?

Reply:
It filters which AWS services are installed, so limiting to just the `dynamodb` service we use to reduce the overall install footprint

http://docs.getmoto.org/en/latest/docs/getting_started.html
https://stackoverflow.com/questions/46775346/what-do-square-brackets-mean-in-pip-install

Reply:
Nevermind, found it at http://docs.getmoto.org/en/latest/docs/getting_started.html